src/Components/Cart/cart.jsx

```diff
@@ -0,0 +1,61 @@
+import { Footer } from "../../Pages/Footer";
+import { Link } from "react-router-dom";
+import { Nav } from "../../Pages/Nav";
+import { CartCard } from "../../Pages/CartCard";
+import { CartPriceDetail } from "../../Pages/CartPriceCard";
+import "../../Styles/cart.css";
+import { useCart } from "../../Context/cart-context";
```

Comment:
suggestion: anything other than a Component should start with a lower case letter.